<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detaildialog</class>
 <widget class="QWidget" name="detaildialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>860</width>
           <height>632</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="titre_global">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>22</pointsize>
                <weight>75</weight>
                <bold>true</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Titre de la photo</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="titre_description">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="titre_infos_bien">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Informations du bien</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="parution_prix">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Parution : 01/01/2016 - Prix : 400 000€</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Informations Contact</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="id_etat">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>id : 1505031006- En Location</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="description">
              <property name="maximumSize">
               <size>
                <width>350</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string>Texte description</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="titre_adresse">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Adresse :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="adresse">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>Adresse, code Postal, ville</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="titre_superficie">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Superficie :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="superficie">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>1000m2</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="titre_pieces">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Nombre de pièces :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="pieces">
                <property name="text">
                 <string>10 pièces</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="titre_prix">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Prix :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="prix">
                <property name="text">
                 <string>400 000€</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="8" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="titre_nomprenom">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Nom, Prénom :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="nom_prenom">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Nom prénom</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="titre_telephone">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Téléphone :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="telephone">
                <property name="text">
                 <string>telephone</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="titre_email">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Email :</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="email">
                <property name="text">
                 <string>email</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0" rowspan="6">
             <layout class="QVBoxLayout" name="vertial_layout_photos">
              <item>
               <widget class="QLabel" name="photo_contractuelle">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Photo contractuelle</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="photos_supplementaires">
                <item>
                 <widget class="QLabel" name="photo1">
                  <property name="text">
                   <string>photo1</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="photo2">
                  <property name="text">
                   <string>photo2</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="photo3">
                  <property name="text">
                   <string>photo3</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="photo4">
                  <property name="text">
                   <string>photo4</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
